server/views/components/formField.njk

```diff
@@ -51,6 +51,8 @@
         <a href="{{ field.value.href }}" class="govuk-link govuk-link--no-visited-state">{{ field.value.text }}</a>
     {% elif field.type === 'every' %}
         {{ field.value | transformValue('every') }}
+    {% elif field.value === true or field.value === false %}
```

Comment:
Need to also check for === 'true'  or === 'false'
If field.value = 'true' or 'false' string. Then it is not formatted Yes/No